lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -172,7 +172,7 @@ const Reportback = {
     const buttonText = Drupal.t('View more');
 
     this.$viewMoreContainer = $(`<div class="reportback__view-more"></div>`);
-    this.$viewMoreButton = $(`<button id="view-more-button" class="button button--view-more -tertiary inline-alt-text-color">${buttonText}</button>`);
+    this.$viewMoreButton = $(`<button id="view-more-button" class="button button--view-more -tertiary inline-alt-text-color ga-click" data-category="Gallery" data-action="click" data-label="View More Clicks">${buttonText}</button>`);
```

Comment:
curious but could we add a `ga` to the new data items to make it more obvious it's for Google Analytics and potentially avoid naming conflicts?